command_line/show_mask_info.py

```diff
@@ -0,0 +1,24 @@
+# LIBTBX_SET_DISPATCHER_NAME dev.dxtbx.show_mask_info
+
+import sys
+
+from dxtbx.model.experiment_list import ExperimentListFactory
+
+
+def show_mask_info():
+    try:
+        el = ExperimentListFactory.from_filenames(sys.argv[1:])
+    except FileNotFoundError as e:
+        sys.exit(str(e))
+
+    for i in el.imagesets():
+        d = i.get_detector()
+        m = i.get_mask(0)
```

Comment:
More verbose variable name here would mean you can use `m` below instead of needing to add an underscore prefix:
```suggestion
        mask = i.get_mask(0)
```

Reply:
I think this is very much a personal choice, and we are talking a couple of lines of code so not too worried by this.